### ISSUE TITLE
Enquiry regarding the license status of the images in your dataset

### ISSUE THREAD

**Issue by layilpaht** [wrkout/exercises.json#305]
Hi,
Thanks for the dataset. I am planning on working on a fitness app & this dataset will contribute a lot in that regards. Thanks again

However I was having a very important doubt regarding the license status of the images in your dataset.
Are the images in your dataset available without any copyright issue/open sourced & can be used in a commercial app without any issues later on?

Thanks & Regards

**Comment by Mathnerd314** [comment]
There is a note [here](https://github.com/wrkout/exercises.json/blob/master/CONTRIBUTING.md#exercise-images):
> Currently all exercises have two images, these have been scraped off the internet, therefore `l do not own the copyright` for these images and would `advise against using them in commercial projects`.

Reverse image search suggests bodybuildlng.com as one source. Quoting [their policy](https://web.archive.org/web/20220501053018/https://bodybuilding.kustomer.help/terms-of-use-Byaiex75D):
> All content, logos, graphics, pages, scripts, and service names included in or made available through any Site are subject to trade dress, trademarks, service marks, and/or copyright law and other laws that protect intellectual property in the U.S. and other countries. Bodybuilding.com's intellectual property may not be used without Bodybuilding.com's express written permission. [...] You may view and use the Content for your personal information and for no other purpose. Bodybuilding.com does not grant to you or any person any right to use, reproduce, copy, modify, transmit, display, publish, sell, license, create derivative works, publicly perform, or distribute by any means, method, or process whatsoever, now known or hereafter developed, any of the Content on or transmitted through the Site.

**Comment by OllieJennings** [comment]
Hey @layilpaht , l have an updated project which has all of this now: https://wrkout.xyz/

Granted it's not free, but its got images and videos of every movement.

**Comment by PranavMasekar** [comment]
@OllieJennings 

Hey, I want to use the images specifically for my app. Is there any way to get a dataset json with all images in it ?? 

I specifically want the json and I don't intent to use any kind of API because my app is completely offline

**Comment by OllieJennings** [comment]
@PranavMasekar, to get the entire dataset would require some pretty substantial one-off payment to be honest.

I currently have a few paying customers and 2 business building upon the API.

**Comment by FreedomChuks** [comment]
@OllieJennings  What images? Are there anatomy image types?

**Comment by OllieJennings** [comment]
> [@OllieJennings](https://github.com/OllieJennings) What images? Are there anatomy image types?

Yes @FreedomChuks as seen here: https://wrkout.xyz/ that look like (watermarked as certain plans):

<img width="720" height="452" alt="Image" src="https://github.com/user-attachments/assets/50b4a44e-720e-4b77-b748-b5b997d63dff" />

**Comment by FreedomChuks** [comment]
@OllieJennings  But there are water marked. I just followed you on LinkedIn, can we discuss more there or do you have any other proper means of communication

**Comment by OllieJennings** [comment]
> [@OllieJennings](https://github.com/OllieJennings) But there are water marked. I just followed you on LinkedIn, can we discuss more there or do you have any other proper means of communication

Sure, they are watermakred at the lower pricing tiers but at the higher they are not

**Comment by DrSocket** [comment]
Hi, how are the images made?

**Comment by OllieJennings** [comment]
@DrSocket They are created by myself using 3D software with skeleton & muscles models, where l create animations and images
